doc/GettingStarted.tex

```diff
@@ -84,14 +85,14 @@ \subsubsection{File}
 
 \item[Output LaTeX] Produce the set of equations that define the
   current system for use in documenting the model, for use in LaTeX
-  compatible typesetting systems. If you LaTeX implement doesn't
+  compatible typesetting systems. If your LaTeX implemention doesn't
   support breqn, untick the \htmlref{wrap long equations
-  option}{wrap-equations}. 
-
+  option}{wrap-equations}, which can be found in the preferences panel under the options menu. 
+ 
 \item[Output MatLab] Output a MatLab function that can be used to
 simulate the system in a MatLab compatible system, such as
-\htmladdnormallinkfoot{MatLab}{https://en.wikipedia.org/wiki/MATLAB}
-or \htmladdnormallinkfoot{Octave}{http://www.gnu.org/software/octave/}.
+\htmladdnormallink{MatLab}{https://en.wikipedia.org/wiki/MATLAB}
```

Comment:
I think it would be better to leave this as htmladdnormallinkfoot, which adds a footnote in the printed version showing the URL.